npm/ux/qsharp-ux.css

```diff
@@ -27,7 +50,16 @@ html {
   padding: 0;
 }
 
-body {
+html {
```

Comment:
Note: The contents of this file haven't actually changed much, mostly just been moved around to group together related styles.